lib/px/pxconfig.lua

```diff
@@ -29,6 +29,7 @@ _M.auth_token = 'PX_AUTH_TOKEN'
 -- _M.proxy_url = nil
 -- _M.proxy_authorization = nil
 -- _M.custom_cookie_header = nil
+-- _M.bypass_monitor_header = nil
```

Comment:
add the new config to the config builder

Reply:
done